Dockerfile

```diff
@@ -77,17 +77,20 @@ RUN cd ./deploy/tools/envs-validator && yarn build
 # ******* STAGE 3: Run ********
 # *****************************
 # Production image, copy all the files and run next
-FROM node:18-alpine AS runner
+FROM node:18-alpine3.18 AS runner
 RUN apk add --no-cache --upgrade bash curl jq unzip
 
 ### APP
 WORKDIR /app
 
+ENV NODE_ENV production
 # Uncomment the following line in case you want to disable telemetry during runtime.
-# ENV NEXT_TELEMETRY_DISABLED 1
+ENV NEXT_TELEMETRY_DISABLED 1
```

Comment:
Is this change necessary? Do you believe that Next.js telemetry can significantly slow down the build process?